HTMLPictureElement listed twice in baselines/dom.generated.d.ts

Found:

* [here](https://github.com/Microsoft/TSJS-lib-generator/blob/b9a172f0eb83f17ca0a6ec60bb6b0051edda95a3/baselines/dom.generated.d.ts#L6664) and 
* [here](https://github.com/Microsoft/TSJS-lib-generator/blob/b9a172f0eb83f17ca0a6ec60bb6b0051edda95a3/baselines/dom.generated.d.ts#L13666)

Is this a bug?

Happy to contribute a PR, but I can't run `build.sh` on my Ubuntu box:

```sh
$ ./build.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   599    0   599    0     0    839      0 --:--:-- --:--:-- --:--:--   838
100 30768  100 30768    0     0  19586      0  0:00:01  0:00:01 --:--:--  117k
Missing method Contains in assembly /home/myitcv/dev/TSJS-lib-generator/.paket/paket.bootstrapper.exe, type System.Linq.Enumerable

Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
File name: 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
File name: 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'

$ mono --version
Mono JIT compiler version 3.2.8 (Debian 3.2.8+dfsg-4ubuntu4)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug
        LLVM:          supported, not enabled.
        GC:            sgen

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 15.10
Release:        15.10
Codename:       wily
```

Worth adding a Travis build to ensure this works for *nix users?

@zhengbli same issue with `4.2.3`:

```
$ mono --version
Mono JIT compiler version 4.2.3 (Stable 4.2.3.4/832de4b Wed Mar 16 13:19:08 UTC 2016)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug
        LLVM:          supported, not enabled.
        GC:            sgen
$ ./build.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   599    0   599    0     0    812      0 --:--:-- --:--:-- --:--:--   811
100 30768  100 30768    0     0  19036      0  0:00:01  0:00:01 --:--:-- 19036

Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
File name: 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
File name: 'System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
```

--- a/baselines/dom.generated.d.ts
+++ b/baselines/dom.generated.d.ts
@@ -3239,8 +3239,6 @@ interface Document extends Node, GlobalEventHandlers, NodeSelector, DocumentEven
       * @param content The text and HTML tags to write.
       */
     writeln(...content: string[]): void;
-    createElement(tagName: "picture"): HTMLPictureElement;
-    getElementsByTagName(tagname: "picture"): NodeListOf<HTMLPictureElement>;
     addEventListener(type: "MSContentZoom", listener: (ev: UIEvent) => any, useCapture?: boolean): void;
     addEventListener(type: "MSGestureChange", listener: (ev: MSGestureEvent) => any, useCapture?: boolean): void;
     addEventListener(type: "MSGestureDoubleTap", listener: (ev: MSGestureEvent) => any, useCapture?: boolean): void;
@@ -3667,7 +3665,6 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     webkitRequestFullscreen(): void;
     getElementsByClassName(classNames: string): NodeListOf<Element>;
     matches(selector: string): boolean;
-    getElementsByTagName(tagname: "picture"): NodeListOf<HTMLPictureElement>;
     addEventListener(type: "MSGestureChange", listener: (ev: MSGestureEvent) => any, useCapture?: boolean): void;
     addEventListener(type: "MSGestureDoubleTap", listener: (ev: MSGestureEvent) => any, useCapture?: boolean): void;
     addEventListener(type: "MSGestureEnd", listener: (ev: MSGestureEvent) => any, useCapture?: boolean): void;
@@ -13658,14 +13655,6 @@ interface ProgressEventInit extends EventInit {
     lengthComputable?: boolean;
     loaded?: number;
     total?: number;
-}
-
-interface HTMLPictureElement extends HTMLElement {
-}
-
-declare var HTMLPictureElement: {
-    prototype: HTMLPictureElement;
-    new(): HTMLPictureElement;
 }
 
 interface ClipboardEventInit extends EventInit {

--- a/inputfiles/addedTypes.json
+++ b/inputfiles/addedTypes.json
@@ -346,34 +346,6 @@
     },
     {
         "kind": "interface",
-        "name": "HTMLPictureElement",
-        "flavor":  "Web",
-        "extends": "HTMLElement",
-        "constructorSignatures": [
-            "new(): HTMLPictureElement"
-        ],
-        "properties": []
-    },
-    {
-        "kind": "method",
-        "interface": "Document",
-        "name": "createElement",
-        "signatures": ["createElement(tagName: \"picture\"): HTMLPictureElement"]
-    },
-    {
-        "kind": "method",
-        "interface": "Document",
-        "name": "getElementsByTagName",
-        "signatures": ["getElementsByTagName(tagname: \"picture\"): NodeListOf<HTMLPictureElement>"]
-    },
-    {
-        "kind": "method",
-        "interface": "Element",
-        "name": "getElementsByTagName",
-        "signatures": ["getElementsByTagName(tagname: \"picture\"): NodeListOf<HTMLPictureElement>"]
-    },
-    {
-        "kind": "interface",
         "flavor": "Web",
         "name": "ClipboardEventInit",
         "extends": "EventInit",

